log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/commons/js/gadget-utils.js

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Hi Thanrik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/55

Thanks,
SajithD